easybuild/easyconfigs/g/gompi/gompi-2016.07.eb

```diff
@@ -0,0 +1,22 @@
+easyblock = "Toolchain"
+
+name = 'gompi'
+version = '2016.07'
+
+homepage = '(none)'
+description = """GNU Compiler Collection (GCC) based compiler toolchain,
+including OpenMPI for MPI support."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+gccver = '6.1.0'
+binutilsver = '2.27'
+gccbinver = '%s-%s' % (gccver, binutilsver)
+
+# compiler toolchain dependencies
+dependencies = [
+   ('GCC', gccbinver),  # includes both GCC and binutils
+   ('OpenMPI', '1.10.2', '', ('GCC', gccbinver)),
```

Comment:
I've changed the version to 1.10.3, I think Open MPI 1.10.2 was the first application I built with EasyBuild (as GCC was handbuilt) and so when making a toolchain I tried to use the versions I already had built - which I know now is not as ideal as using the tried and tested combinations. I just did a build of 1.10.3 and committed it so it should be okay now.